workspaces/analytics/.changeset/renovate-6b4545b.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-analytics-module-matomo': patch
+'@backstage-community/plugin-analytics-provider-segment': patch
+---
+
+Updated dependency `@types/node` to `22.14.1`.
```

Comment:
Sorry, somewhat confused, I don't see any chance for this just the changeset? Or were these already merged and this is to release those changes?

Reply:
Ah, leftover from prior state - removed, thanks!